templates/webapi/comments/add_comment_form_groups.html.ep

```diff
@@ -52,6 +52,9 @@
                 % } else {
                     <a class="help_popover fa fa-ticket" title="Add force result label" role="button"
                        data-template="label:force_result:new_result[:description_or_bugref]" onclick="insertTemplate(this)"></a>
+                    <br>
+                    <a class="help_popover fa fa-flag" title="Flag this comment for carryover without bugref (or bugref only in force-result label)" role="button"
```

Comment:
```suggestion
                    <a class="help_popover fa fa-flag" title="Flag this comment for carryover without issue reference (or issue reference only in force-result label)" role="button"
```

But I don't understand the part "(or issue reference only in force-result label)". I suggest you make it a separate complete sentence.